src/katgpucbf/fgpu/send.py

```diff
@@ -234,10 +236,11 @@ def make_streams(
     differ only in the network interface used (there is one per interface).
     Thus, they can be used interchangeably for load-balancing purposes.
     """
-    dtype = chunks[0].data.dtype
+    dtype = chunks[0].data.dtype  # Type for each complex value
+    bandwidth = adc_sample_rate * 0.5
```

Comment:
I'm not sure how else to ask this, but
* Where did this come from?

Reply:
Let's start with wideband. The formula for rate should always have been bandwidth * N_POLS * COMPLEX * bytes-per-real-component. It just so happens that `bandwidth * COMPLEX` equals adc_sample_rate, which is why it was correct before when the `dtype` was per-real-component. The `dtype` is now per-complex-value, so COMPLEX is now absorbed into `dtype.itemsize`.

For narrowband, this is actually wrong now (will fix). It should be
```suggestion
    bandwidth = adc_sample_rate * 0.5 * send_rate_factor
```
and then `send_rate_factor` will no longer appear in computing `rate`.

Reply:
Excellent, I see the latest commits address my counter-question of 
> Will `send_rate_factor` be removed?